### ISSUE TITLE
RBAC- Read Only Access for Service Desk when manaing Cluster through WAC

### ISSUE THREAD

**Issue by virenderrawatgit** [MicrosoftDocs/Windows-Admin-Center-Ideas-and-Feedback#289]
WAC is missing the RBAC Access feature when managing Cluster from WAC.
We have no option where we can give Read Only access to our service Desk Team over WAC so they can atleast monitor the cluster health through WAC.

We should have some roles which we can give to different groups so they can access them as per their requirements.
administrator >> full Admin and manage cluster through WAC
Service Desk >> who can view the cluster resource through WAC
VM team >> who can deploy, delete, reboot, etc through WAC

**Comment by AnandPrakashSharma** [comment]
Yes Virender is this right, We should have such types of groups to separate the user's permission. Due to the lack of these permissions, Customer is looking for different tools as well like SCVMM. Team should think and try to add this ASAP.

**Comment by RebeccaMbula** [comment]
Hi, We are doing a deep investigation into RBAC and this feedback is extremely useful to us! Thank you, we will keep this in mind when doing development work.

**Comment by RebeccaMbulaMs** [comment]
Hi @virenderrawatgit and @AnandPrakashSharma 
I'd like to hear your requirements for RBAC for your current environment. 
Please ping me on rwambua@microsoft.com and lets set a call?
